packages/react-components/source/react/library/sidepanel/SidePanel.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '../button';
+import Heading from '../heading';
+
+const propTypes = {
+  className: PropTypes.string,
+  children: PropTypes.node,
+  contentClassName: PropTypes.string,
+  onClose: PropTypes.func,
+  open: PropTypes.bool,
+  title: PropTypes.string,
+  type: PropTypes.oneOf(['toolbar']).isRequired,
+};
+
+const defaultProps = {
+  children: null,
+  open: true,
+  title: '',
+  onClose() {},
+  className: '',
+  contentClassName: '',
+};
+
+/**
+ * SidePanel may be used inside the rightmost Columns.Column.
+ */
+const SidePanel = ({
+  children,
+  open,
+  title,
+  onClose,
+  className,
+  contentClassName,
+}) => {
+  const actions = (
+    <Button
+      icon="close"
```

Comment:
I'm actually not sure why we have both of these, but I think the icon should be `icon="x"`